web/source/resources/osk/kmwosk.css

```diff
@@ -333,21 +383,21 @@ div.ios div.kmw-keytip {
     color:#fff;
```

Comment:
Does the iOS user test need to check that key tips are styled fine for dark mode?

Reply:
Yes, added a darkmode user test, thanks.